airflow/contrib/operators/emr_create_job_flow_operator.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from airflow.contrib.hooks.emr_hook import EmrHook
+from airflow.models import BaseOperator
+from airflow.utils import apply_defaults
+from airflow.exceptions import AirflowException
+
+
+class EmrCreateJobFlowOperator(BaseOperator):
+    """
+    Creates an EMR JobFlow, reading the config from the EMR connection.
+    A dictionary of JobFlow overrides can be passed that override the config from the connection.
+
+    :param aws_conn_id: aws connection to uses
+    :type aws_conn_id: str
+    :param emr_conn_id: emr connection to use
+    :type emr_conn_id: str
+    :param job_flow_overrides: boto3 style arguments to override emr_connection extra
+    :type steps: dict
+    """
+    template_fields = []
+    template_ext = ()
+    ui_color = '#f9c915'
+
+    @apply_defaults
+    def __init__(
+            self,
+            aws_conn_id='s3_default',
+            emr_conn_id='emr_default',
+            job_flow_overrides={},
```

Comment:
hmm, that's good to know, thanks. i'm pretty new to python so it's good to learn its quirks.